bugbug/model.py

```diff
@@ -248,10 +248,7 @@ def get_top_features(cutoff, shap_values):
 
             class_name = self.le.inverse_transform([num])[0]
 
-            important_features["classes"][class_name] = (
-                top_item_features,
-                top_avg,
-            )
+            important_features["classes"][class_name] = (top_item_features, top_avg)
```

Comment:
This change seems unrelated, could you revert it?

Reply:
pre-commit seems to be making this change, not sure why it is detecting this

Reply:
You should be able to revert it now, it was probably a different version of black